x-pack/plugins/observability_solution/slo/public/locators/slo_edit.ts

```diff
@@ -29,9 +29,7 @@ export class SloEditLocatorDefinition implements LocatorDefinition<SloEditLocato
           ...slo,
         },
         { useHash: false, storeInHashQuery: false },
-        slo.id
-          ? `${SLO_PREFIX}/edit/${encodeURIComponent(String(slo.id))}`
-          : `${SLO_PREFIX}${SLO_CREATE_PATH}`
```

Comment:
@kdelemme I searched for `SLO_PREFIX` in the slo folder and it looks like it is used also in the `slo_details` and `slo_list` locators as well. I will remove from there as well and will get rid of SLO_PREFIX since it is not used anywhere else